src/sentry/event_manager.py

```diff
@@ -178,6 +179,14 @@ def is_sample_event(job):
     return get_tag(job["data"], "sample_event") == "yes"
 
 
+def normalized_sdk_tag_from_event(event: Event) -> str:
```

Comment:
It would be good to document why we need to use `normalized_sdk_tag_from_event` (e.g. it reduces the cardinality and cost associated with DD)

Reply:
added